modules/run-vault/run-vault

```diff
@@ -443,13 +468,15 @@ function run {
   local systemd_stderr=""
   local user=""
   local skip_vault_config="false"
-  local enable_s3_backend="false"
+  local storage_backend="consul"
+  local ha_storage_backend="consul"
```

Comment:
NIT: Store these values in `readonly` variables at the top. Use the variables here and in the docs.